test/e2e/storage/testsuites/multivolume.go

```diff
@@ -334,6 +350,14 @@ func (t *multiVolumeTestSuite) DefineTests(driver TestDriver, pattern testpatter
 		if l.config.ClientNodeName != "" {
 			framework.Skipf("Driver %q requires to deploy on a specific node - skipping", l.driver.GetDriverInfo().Name)
 		}
+		// For multi-node tests there must be enough nodes with the same toopology to schedule the pods
```

Comment:
This logic is the same in many tests. Can we consolidate all the "different-node test requirement" checks into a common function?

Reply:
Most of the tests have a lot of code in common. I just followed the existing pattern. But yes, I will move the check to a function.

Reply:
I think the whole section starting from "// Check different-node test requirement" could be consolidated to a common method. But it doesn't have to be in this pr.